packages/dev/loaders/src/glTF/2.0/Extensions/MSFT_lod.ts

```diff
@@ -18,7 +18,7 @@ declare module "../../glTFFileLoader" {
         /**
          * Defines options for the MSFT_lod extension.
          */
-        [NAME]?: Partial<{
+        ["MSFT_lod"]?: Partial<{

```

Comment:
If name is not supported, should it not be changed in all gltf extensions ?

Reply:
MSFT_lod is the only extension with options.